diode-server/reconciler/server.go

```diff
@@ -109,6 +109,12 @@ func (s *Server) RetrieveIngestionDataSources(_ context.Context, in *reconcilerp
 	return &reconcilerpb.RetrieveIngestionDataSourcesResponse{IngestionDataSources: dataSources}, nil
 }
 
+// RetrieveIngestionLogs retrieves logs
+func (s *Server) RetrieveIngestionLogs(_ context.Context, _ *reconcilerpb.RetrieveIngestionLogsRequest) (*reconcilerpb.RetrieveIngestionLogsResponse, error) {
+	logs := make([]*reconcilerpb.IngestionLog, 0)
```

Comment:
@mfiedorowicz do we need API key for logs as well?

Reply:
Yes, `NETBOX_TO_DIODE` API key please